src/pretalx/schedule/models/slot.py

```diff
@@ -27,9 +27,10 @@ class TalkSlot(LogMixin, models.Model):
     is_visible = models.BooleanField()
     start = models.DateTimeField(null=True)
     end = models.DateTimeField(null=True)
+    available_index = models.PositiveIntegerField(default=0)
```

Comment:
Why did you add this field?

Reply:
i think this field is needed to uniquely identify the slots for the submission  -
its used in the update_talk_slots to update/create or delete the slots...